### PR TITLE
#1117: Addressed review comments

### PR DESCRIPTION
**Pull request by apoorvyadav1111**
Hi team,

This PR is created to address few comments left on #1117 and to address those on priority

### CODE REVIEW

File: internal/eval/eval.go [hunk 1]
```diff
@@ -54,6 +54,16 @@ type EvalResponse struct {
...
+// These ensure that result and error are mutually exclusive.
+// If result is nil, then error should be non-nil and vice versa.
+
+// makeEvalResult creates a new EvalResponse with the given result and nil error.
+// This is a helper function to create a new EvalResponse with the given result and nil error.
+/**
+ * @param {interface{}} result - The result of the store operation.
+ * @returns {EvalResponse} A new EvalResponse with the given result and nil error.
+ */
 //go:inline
```
Comment (JyotinderSingh) [inline internal/eval/eval.go:67]: Let's remove this directive (it anyway does nothing on newer versions of Go)

File: internal/eval/eval.go [hunk 2]
```diff
@@ -62,6 +72,12 @@ func makeEvalResult(result interface{}) *EvalResponse {
 	}
 }
 
+// makeEvalError creates a new EvalResponse with the given error and nil result.
+// This is a helper function to create a new EvalResponse with the given error and nil result.
+/**
+ * @param {error} err - The error that occurred during the store operation.
+ * @returns {EvalResponse} A new EvalResponse with the given error and nil result.
+ */
 //go:inline
```
Comment (JyotinderSingh) [inline internal/eval/eval.go:81]: Let's remove this directive

**Review by JyotinderSingh — COMMENTED** [review]
Little unrelated but we should remove the unnecessary inline directives. The compiler is smart enough to judge inlining requirements.
Additionally, newer versions of go don't really care about these directives.

File: internal/eval/eval.go (hunk 1 above)
Comment (apoorvyadav1111) [reply internal/eval/eval.go]: Hi, thanks for the review, removed in the latest commit.

File: internal/eval/eval.go (hunk 2 above)
Comment (apoorvyadav1111) [reply internal/eval/eval.go]: done sir!

**Review by lucifercr07 — APPROVED** [review]

**Pull request merged**